controllers/hibp.js

```diff
@@ -8,6 +8,9 @@ const sha1 = require("../sha1-utils");
 const HBSHelpers = require("../hbs-helpers");
 
 async function notify (req, res) {
+  if(!["breachName", "hashPrefix", "hashSuffixes"].every(req.body.hasOwnProperty, req.body)) {
+    throw new Error("breach notification requires breachName, hashPrefix, and hashSuffixes");
```

Comment:
ultra nit: the other error messages (in this PR, not sure about other errors) have a period at the end of the sentence. 🤷‍♂️
Also not sure re: "breach" vs "Breach".